CrashLib/CrashLib/MSCrashCustomView.m

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSCrashCustomView.h"
+
+#if TARGET_OS_OSX
+#import <Cocoa/Cocoa.h>
+#else
+#import <UIKit/UIKit.h>
+#endif
+
+#if TARGET_OS_OSX
+@interface MSCustomView : NSView
+#else
+@interface MSCustomView : UIView
+#endif
+
+@end
+
+@implementation MSCustomView
+
+#if TARGET_OS_OSX
+-(void)drawRect:(NSRect)rect {
+#else
+-(void)drawRect:(CGRect)rect {
+#endif
+    [super drawRect:rect];
+    @throw [NSException exceptionWithName:NSGenericException reason:@"Crash from drawing custom view."
```

Comment:
This one doesn't crash on iOS and tvOS, if that's expected we should document it.

Reply:
Fixed